src/web3py/extensions/contracts.py

```diff
@@ -17,6 +17,8 @@
 
 
 class LidoContracts(Module):
+    w3: Web3
```

Comment:
For linters.
New version of web3 modules contains smth like
w3: Unify[Web3, AsyncWeb3]